src/Apps/Auction/Components/BidForm.tsx

```diff
@@ -35,19 +36,42 @@ const validationSchema = Yup.object().shape({
   ),
 })
 
+const getSelectedBid = ({
+  initialSelectedBid,
+  displayIncrements,
+}: {
+  initialSelectedBid: Props["initialSelectedBid"]
+  displayIncrements: Array<{ value: string; text: string }>
+}): string => {
+  if (!initialSelectedBid) {
+    return displayIncrements[0].value
+  } else {
+    const selectedNum = Number(initialSelectedBid)
+    const lastGoodIncrement = findLast(
+      displayIncrements,
+      i => Number(i.value) === selectedNum
+    )
+    return (lastGoodIncrement || displayIncrements[0]).value
```

Comment:
This is #superminor DX thing but it's often a good habit to assign the return value to a variable (and return that), so that the variable can be logged later on down the line by future devs or a breakpoint inserted inbetween.